src/applications/edu-benefits/0994/containers/EducationGate.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import RequiredLoginView from '../../../../platform/user/authorization/components/RequiredLoginView';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
+
+export function EducationGate({ user, location, children }) {
+  // Short-circuit the check on the intro page
+  if (location.pathname === '/introduction') {
+    return children;
+  }
+
+  if (
+    user.login.currentlyLoggedIn &&
```

Comment:
Does this form require you to be logged in? Just wanted to check because most of our forms do not require login.

Reply:
It's in the AC of https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16434